documentation/subcommands.markdown

```diff
@@ -19,98 +18,95 @@ The following subcommands are provided:
 
 The format for each subcommand is:
 
-```sh
+~~~sh
 puppetserver <subcommand> [<args>]
-```
+~~~
 
 When running from source, the format is:
 
-```sh
+~~~sh
 lein <subcommand> -c /path/to/puppet-server.conf [--] [<args>]
-```
+~~~
 
-Note that if one of the `<args>` begins with a `-` (like `--version` or `-e`)
-then the intermediate `--` above is required for the argument to be applied to
-the subcommand and not leiningen. This is not necessary when running from
-packages (i.e. `puppetserver <subcommand>`).
+Note that if you are running from source, you need to separate flag arguments (such as `--version` or -`e`) with `--`, as shown above. Otherwise, those arguments will be applied to Leiningen instead of to Puppet Server. This isn't necessary when running from
```

Comment:
Cosmetic - seems like the "-" before `e` should be inside of the accent characters - like `-e`.